README.md

```diff
@@ -115,18 +115,18 @@ It consists of
 
 ## Reference
 
-[1] Ferragina, P., & Manzini, G. (2000). Opportunistic data structures with
+[^1]: Ferragina, P., & Manzini, G. (2000). Opportunistic data structures with
```

Comment:
Let's use the footnote syntax for other references [2], [3], [4], like what we will do for `lib.rs`

Reply:
Oh yeah I see I missed it!

Reply:
Done now. I think this is ready for merging.